pkg/extensions/tbtc/recovery/electrs.go

```diff
@@ -0,0 +1,77 @@
+package recovery
```

Comment:
Maybe this code should go into `pkg/chain/bitcoin` as we already got bitcoin specific config there?